lib/std/build.zig

```diff
@@ -2451,6 +2454,14 @@ pub const LibExeObjStep = struct {
                 zig_args.append("--version") catch unreachable;
                 zig_args.append(builder.fmt("{}", .{version})) catch unreachable;
             }
+
+            const install_name = self.install_name orelse builder.fmt("@rpath/{s}{s}{s}", .{
```

Comment:
This should only be emitted when the target is darwin, yeah?

Reply:
Good catch, thanks!